v2/pkg/runner/targets.go

```diff
@@ -44,7 +44,14 @@ func (r *Runner) mergeToFile() (string, error) {
 
 	// target defined via CLI argument
 	if r.options.Host != "" {
-		fmt.Fprintf(tempInput, "%s\n", r.options.Host)
+		if strings.Contains(r.Options.Host, ","){
```

Comment:
I think this could be simplified to:
```go
if r.options.Host != "" {
	for _, host := range strings.Split(r.options.Host, ",") {
		fmt.Fprintf(tempInput, "%s\n", host)
	}
}
```